src/test/java/org/opentripplanner/updater/trip/TimetableSnapshotSourceTest.java

```diff
@@ -154,6 +153,8 @@ public void testHandleCanceledTrip() throws InvalidProtocolBufferException {
       feedId
     );
 
+    updater.commitTimetableSnapshot(true);
```

Comment:
If you change the refresh rate to zero in the updater you probably don't need this?

Reply:
Oh god, you're right. I totally missed the fact that this one test uses its own method to instantiate the snapshot source.